code_gen_config.yml

```diff
@@ -4,7 +4,7 @@ extra_header: |
   // Copyright (C) 2017-2024 Adam Lock, Einar Omang
 targets:
   - type: types
-    file_path: schemas/1.0.4/Opc.Ua.Types.bsd
+    file_path: schemas/1.05/Opc.Ua.Types.bsd
```

Comment:
Shouldn't this be 1.0.5?

Reply:
It's actually called 1.05, the version of the standard, that is. Either way this is just the path in the repo.